server/build-plugins.js

```diff
@@ -124,15 +124,7 @@ async function symlinkLocal(packageName) {
 async function checkForModuleExistence(pkg) {
   const pkgPath = resolve(MODULES_DIRECTORY, pkg);
   const exists = fs.existsSync(pkgPath);
-
```

Comment:
Was this not doing anything useful?

Reply:
Yeah, good question. This was causing problems where `npm install` would be run if there was a local version of a plugin because they are symlinked which wasn't the behavior we were looking for. I couldn't really tell what this was originally for but my guess was it was left over from when I was trying to optimize the local plugin flow. The old logic returns true if the folder exists in `node_modules` and is not a symbolicLink, which would cause `npm install` to run and overwrite the local symlinked module which we don't want.